sql/updates/auth/6.x/9999_99_99_18010_auth.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `rbac_permissions` WHERE `id` IN (851, 852);
+INSERT INTO `rbac_permissions` (`id`, `name`) VALUES
+(851, 'Command: .reload conversation_template'),
+(852, 'Command: .debug conversation');
```

Comment:
Thanks, I will fix that too. Forgot to change it outside of solving the conflict.